f2/src/utils/__tests__/formatAnalystEmail.test.js

```diff
@@ -8,4 +8,14 @@ describe('formatAnalystEmail', () => {
     expect(s).toEqual(expect.stringMatching(/ERROR/))
     expect(console.error).toHaveBeenCalled()
   })
+
+  it('flags self harm words prominently if present', () => {
+    const data = { selfHarmWords: 'agile' }
```

Comment:
👀

Reply:
> 👀

The test uses a fake list of self harm words, as CAFC didn't want up to put the real list in the repo.